_automating-configurations/api/deprovision-workflow.md

```diff
@@ -9,7 +9,9 @@ nav_order: 70
 
 When you no longer need a workflow, you can deprovision its resources. Most workflow steps that create a resource have corresponding workflow steps to reverse that action. To retrieve all resources currently created for a workflow, call the [Get Workflow Status API]({{site.url}}{{site.baseurl}}/automating-configurations/api/get-workflow-status/). When you call the Deprovision Workflow API, resources included in the `resources_created` field of the Get Workflow Status API response will be removed using a workflow step corresponding to the one that provisioned them.
 
-The workflow executes the provisioning workflow steps in reverse order. If failures occur because of resource dependencies, such as preventing deletion of a registered model if it is still deployed, the workflow attempts retries.
+The workflow executes the provisioning workflow steps in reverse order. If failures occur because of resource dependencies, such as preventing deletion of a registered model if it is still deployed, the workflow retries as long as at least one resource was deleted.
+
+To prevent data loss, resources created with `create_index`, `create_search_pipeline`, and `create_ingest_pipeline` steps require the resource ID to be included in the `allow_delete` parameter.
```

Comment:
Need to add documentation for the 3 new delete steps to [this page](https://opensearch.org/docs/latest/automating-configurations/workflow-steps/).

Also that would need to link to "Delete Search Pipeline" API which is [not documented](https://opensearch.org/docs/latest/search-plugins/search-pipelines/creating-search-pipeline/)

@kolchfa-aws should I just submit the missing delete search pipeline API with this PR (for 2.16) or submit a separate PR since the API has been there since 2.7.0?

Reply:
@dbwiddis Submitting with this PR is fine. We don't support earlier documentation versions. Thanks!